cmds/bmp/sl_command.go

```diff
@@ -45,10 +50,102 @@ func (cmd slCommand) Options() cmds.Options {
 
 func (cmd slCommand) Validate() (bool, error) {
 	cmd.printer.Printf("Validating %s command: options: %#v", cmd.Name(), cmd.options)
+
+	if !cmd.options.Packages && cmd.options.PackageOptions == "" {
```

Comment:
I changed cmd.options.Packages from string to bool, so when execute "bmp sl --packages", all packages will be listed.